src/Scheduling/MassTransit.QuartzIntegration/MassTransitJobFactory.cs

```diff
@@ -18,12 +18,16 @@ public class MassTransitJobFactory :
         IJobFactory
     {
         readonly IBus _bus;
+        readonly IJobFactory _jobFactory;

```

Comment:
_innerFactory might communicate the decorator pattern better

Reply:
it could be better, but I also kept in mind the case when you provide no factory, and default one can't create MT related jobs. So it is mostly chain of responsibility, not a decorator.
Decorator was my initial idea.